### DISCUSSION TITLE
ESP32-cam motion detection

### DISCUSSION THREAD

**Discussion by aug70** [arendst/Tasmota#16749] [Q&A]
Is it possible to detect motion with esp32-cam? Maybe there are some Tasmota rules that can help?
I want to detect motion with the camera to trigger actions on HomeAssistant.

**Comment by sfromis** [comment]
Not tried using it, but I see that the [Scripting Language](https://tasmota.github.io/docs/Scripting-Language/) feature docs page mentions 
> `res=wc(6 p)` start stop motion detector, p=0 => stop detector, p=T start detector with picture every T ms, -1 get picture difference, -2 get picture brightness

**Comment by gemu2015** [comment]
check this https://github.com/arendst/Tasmota/discussions/16074
